src/test/scala/com/target/data_validator/ValidatorTableSpec.scala

```diff
@@ -183,8 +183,44 @@ class ValidatorTableSpec extends FunSpec with Matchers with TestingSparkSession
 
       }
 
-    }
+      describe("should work for the unique parts of ValidatorCustomFormat") {
```

Comment:
Can we have a test that uses a built-in like the `"json"` format to load source controlled data + verify end-to-end?

Reply:
☝️ #86